.pipelines/build-and-push-images.yml

```diff
@@ -15,13 +23,30 @@ jobs:
   - template: ./templates/template-az-cli-login.yml
     parameters:
       azureDevOpsJSONSPN: $(aro-v4-ci-devops-spn)
+
+
   - template: ./templates/template-push-images-to-acr.yml
     parameters:
       rpImageACR: $(RP_IMAGE_ACR)
+
   - template: ./templates/template-az-cli-logout.yml
   - script: |
       cp -a --parents aro "$(Build.ArtifactStagingDirectory)"
     displayName: Copy artifacts
   - task: PublishBuildArtifacts@1
     displayName: Publish Artifacts
     name: aro_deployer
+
+  - script: |
+      set -xe
+      MESSAGE="$(git for-each-ref refs/tags/$TAG --format='%(contents)')"
+      echo "$MESSAGE" > $(System.DefaultWorkingDirectory)/release.txt
+    displayName: Extract release notes from annotated tag
+  - task: GithubRelease@0
```

Comment:
Release task generating the automated changelog in Github/Releases. It require service connection of type Github to Azure/ARO-RP granted by access token created at https://github.com/settings/tokens with write permission to the ARO-RP repository.